composer.json

```
@@ -3,9 +3,9 @@
     "description": "A non-blocking ElasticSearch client for PHP based on Amp.",
     "license": "MIT",
     "require": {
-        "php": "~7.1.0|~7.2.0|~7.3.0|~7.4.0",
+        "php": ">=7.3",
```

This means that this package should work even with PHP 9 for example...
I'd prefer to limit the supported PHP versions to what we run in CI.

```suggestion
        "php": "~7.3.0|~7.4.0|~8.0.0",
```